net/libfab/libfab.c

```diff
@@ -1310,6 +1311,8 @@ static int libfab_passive_ep_create(struct m0_fab__ep *ep,
 	hints->fabric_attr->prov_name = NULL;
 	tm->ftm_fab->fab_fi = fi;
 	tm->ftm_fab->fab_prov = idx;
+	tm->ftm_fab->fab_max_iov = min32u(fi->tx_attr->iov_limit,
```

Comment:
We can move the log on 1308-1310 after this and add this param fab_max_iov to debug in case if this value is going lower

Reply:
This is getting logged in libfab_bulk_op hence no need to log it here.